### PR TITLE
Add support for only installing and running the schema installer job

### PR DESCRIPTION
**Pull request by rubenvp8510**
This address BZ https://bugzilla.redhat.com/show_bug.cgi?id=1632869

### CODE REVIEW

**Comment by rubenvp8510** [comment]
@vrutkovs well, yes I think it would be. it would be more clear how to use it.

**Comment by rubenvp8510** [comment]
@vrutkovs I did the changes, created another playbook just for install the schema job.

**Comment by rubenvp8510** [comment]
@vrutkovs This is ready for another review. Thanks!

File: roles/openshift_metrics/tasks/create_temp_dirs.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  file: path={{mktemp.stdout}}/templates state=directory mode=0755
+  changed_when: False
+  when: metrics_create_template_dir | bool
+
+- name: Create temp directory local on control node
+  local_action: command mktemp -d
+  register: local_tmp
+  changed_when: False
+  vars:
+    ansible_become: false
```
Comment (vrutkovs) [inline roles/openshift_metrics/tasks/create_temp_dirs.yaml]: Not sure if this is necessary, but its fine to keep it

Comment (rubenvp8510) [reply roles/openshift_metrics/tasks/create_temp_dirs.yaml]: not sure either, but I want to preserve the metrics config.yaml playbook as much as possible.

**Comment by rubenvp8510** [comment]
@vrutkovs not sure why this is failing, but it seems like is not related to this PR, :( 

`The error says: unable to connect to the server: dial tcp 35.196.178.39:8443: connect: no route to host\nerror:`

**Pull request merged**